config/database.yml

```diff
@@ -50,4 +51,4 @@ test:
 #
 production:
   <<: *default
-  database: qa_project_production
+  database: QA_Project
```

Comment:
checkout toàn bộ file này
sửa môi trường của e sao cho chạy dc project nhé

Reply:
vang a